schema_based_on_diagram.sql

```diff
@@ -0,0 +1,49 @@
+CREATE DATABASE clinic;
+
+CREATE TABLE patients (
+    id BIGSERIAL NOT NULL PRIMARY KEY,
+    name VARCHAR(100) NOT NULL,
+    date_of_birth DATE NOT NULL
+);
+
+CREATE TABLE medical_histories (
+    id BIGSERIAL NOT NULL PRIMARY KEY,
+    admitted_at timestamp,
+    patient_id BIGINT REFERENCES patients (id)
```

Comment:
- [ ] It would be better if we address the comment made by the previous code reviewer about the `status` column. I will however, highlight on it for more clarity. Adding a status column will make our medical_histories table look like this
```

CREATE TABLE medical_histories (
    id BIGSERIAL NOT NULL PRIMARY KEY,
    admitted_at timestamp,
    patient_id BIGINT REFERENCES patients (id)
    status varchar(20)
)

```